### PR TITLE
fdclone 3.01d

### PR DESCRIPTION
**Pull request by sanadan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fdclone.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Fdclone < Formula
   desc "Console-based file manager"
   homepage "https://hp.vector.co.jp/authors/VA012337/soft/fd/"
-  url "https://hp.vector.co.jp/authors/VA012337/soft/fd/FD-3.01b.tar.gz"
-  sha256 "d66d902cac9d4f64a91d42ceb487a138d544c9fd9cb2961730889cc8830303d4"
+  url "http://ftp.vector.co.jp/69/81/1362/FD-3.01d.tar.gz"
```
Comment (DomT4) [inline Formula/fdclone.rb]: What's with the protocol downgrade here?

File: Formula/fdclone.rb [hunk 2]
```diff
@@ -40,4 +40,7 @@ def caveats; <<~EOS
         MESSAGELANG="ja"
     EOS
   end
+
+  test do
```
Comment (DomT4) [inline Formula/fdclone.rb]: Actually adding a test rather than just a test block would be good 😜.

File: Formula/fdclone.rb (hunk 1 above)
Comment (ilovezfs) [reply Formula/fdclone.rb]: I think we'll have to ask upstream. Apparently, this isn't even the canonical URL, which is actually http://www.unixusers.net/src/fdclone/FD-3.01d.tar.gz

**Comment by sanadan** [comment]
Thank you. I will try to fix it.

**Pull request closed without merging**